### PR TITLE
feat(sandbox): using given port

### PR DESCRIPTION
**Pull request by mapleeit**
- [x] add unit test

### CODE REVIEW

File: jina/hubble/hubio.py [hunk 1]
```diff
@@ -613,19 +613,26 @@ def deploy_public_sandbox(uses: str):
 
         host = None
         try:
-            res = requests.get(
+            json = requests.get(
```
Comment (JoanFM) [inline jina/hubble/hubio.py:616]: can we have a different name, it can be confused with the package

Comment (JoanFM) [reply jina/hubble/hubio.py:616]: maybe json_response?

**Review by JoanFM — CHANGES_REQUESTED** [review]
There should be some tests regarding this behavior

File: jina/hubble/hubio.py (hunk 1 above)
Comment (mapleeit) [reply jina/hubble/hubio.py]: addressed

**Comment by mapleeit** [comment]
Good job @delgermurun 

The tests have been added, I think it's ready to go, now. @JoanFM

**Review by JoanFM — APPROVED** [review]

**Pull request merged**